### PR TITLE
modify DG645 #2

### PR DESCRIPTION
**Pull request by Lagikna**
- [ ] This change has been discussed with the author.
- [ ] It implements / fixes issue #.

---

**Description:**

The delays' unit of input and output was converted to **'us'**.

### CODE REVIEW

File: drivers/DG645.py [hunk 1]
```diff
@@ -59,6 +59,20 @@ def performGetValue(self, quant, **kw):
...
             return super(Driver, self).performGetValue(quant, **kw)
+
+    def performSetValue(self, quant,value, **kw):
+        set_Delays = ['T0 Length','AB Delay','AB Length',
+            'CD Delay','CD Length',
+            'EF Delay','EF Length',
+            'GH Delay','GH Length'
+        ]
+        if quant.name in set_Delays and quant.set_cmd is not '':
+            value=value/1e6  #convert to us
```
Comment (feihoo87) [inline drivers/DG645.py]: This convert unit from us to s

**Review by feihoo87 — COMMENTED** [review]
Why use `us` rather than `s`?

File: drivers/DG645.py (hunk 1 above)
Comment (Lagikna) [reply drivers/DG645.py]: 比如我输入的value=10，实际传递进去的值是10e-6，仪器接收是按秒为单位的，所以我输入是微秒单位

Comment (Lagikna) [reply drivers/DG645.py]: 所以是没错的

**Comment by Lagikna** [comment]
用秒输入不方便，比如0.000005，us是比较合适的单位

**Comment by feihoo87** [comment]
Use `us` as the default unit confuse me. I think it is the better choice to send an argument `unit` to the `getValue` method. Like these:
```python
ins.setValue('GH Delay', 0.1)             # in unit s
ins.setValue('GH Delay', 0.1, unit='us')  # in unit us
```
I am considering to add `unit` argument support.

**Pull request merged**